namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -100,432 +95,335 @@ public GroupManagerDao(Logger logger, Database db){
 		
 		// This lists all unique supergroups (names) which are parent(s) for all instances (name/id pairs) of the subgroup. 
 		// I expect most implementations to ignore if this has multiple results; a "safe" implementation will check.
-		getSuperGroup ="SELECT DISTINCT f.group_name FROM faction_id f "
+	public static final String getSuperGroup ="SELECT DISTINCT f.group_name FROM faction_id f "
 				+ "INNER JOIN faction_id sf ON sf.group_name = ? "
 				+ "INNER JOIN subgroup sg ON sg.group_id = sf.group_id "
 				+ "WHERE f.group_id = sg.sub_group_id";
 		
 		// returns count of unique names, but not (name / id pairs) of all groups.
-		countGroups = "select count(DISTINCT group_name) as count from faction";
+	public static final String countGroups = "select count(DISTINCT group_name) as count from faction";
 		
 		// returns count of unique names of groups owned by founder
-		countGroupsFromUUID = "select count(DISTINCT group_name) as count from faction where founder = ?";
+	public static final String countGroupsFromUUID = "select count(DISTINCT group_name) as count from faction where founder = ?";
 		
-		mergeGroup = "call mergeintogroup(?,?)";
+	public static final String mergeGroup = "call mergeintogroup(?,?)";
 		
-		updatePassword = "update faction set `password` = ? "
+	public static final String updatePassword = "update faction set `password` = ? "
 				+ "where group_name = ?";
 		
-		updateOwner = "update faction set founder = ? "
+	public static final String updateOwner = "update faction set founder = ? "
```

Comment:
Any specific reason those are all public. Not much of an issue, because they are final, but I don't see any situation where this string would be needed outside of the class.